ofrak_core/ofrak/core/instruction.py

```diff
@@ -176,7 +177,7 @@ async def modify(self, resource: Resource, config: InstructionModifierConfig):
             len(asm) == resource_memory_region.size
         ), "The modified instruction length does not match the original instruction length"
 
-        new_attributes = Instruction.attributes_type(  # type: ignore
+        new_attributes = AttributesType[Instruction](  # type: ignore
```

Comment:
Same question as above (and throughout): is it still necessary to type ignore here?